vars/main.yml

```diff
@@ -1,11 +0,0 @@
----
-
-# Default WATO host attributes. This variable, when being merged with
-# `checkmk_agent__host_attributes`, allows to successfully compare the WebAPI
-# get_host() response in case the `contactgroups` key is set.
```

Comment:
Hmn, not so happy that you remove this from here. This is something that for developer convenience and readability has been separated into an own (internal) variable. However, this must not be changed by the user. That's why I tried to "hide" it in the `vars` file.